src/lib/containers/entity/annotations/CustomTextWidget.tsx

```diff
@@ -0,0 +1,22 @@
+import { Widget, WidgetProps } from '@sage-bionetworks/rjsf-core'
+import React from 'react'
+
+/**
+ * This TextWidget is almost identical to the rjsf TextWidget, except we handle numbers like strings to prevent issues if
+ * the actual annotation data is not a string.
+ */
```

Comment:
Without this custom widget, non-number annotations that correspond to number fields will be auto-converted to 'undefined' without the user's knowledge.